### PR TITLE
Avoid hardcoding branding in user onboarding

### PR DESCRIPTION
**Pull request by justjanne**
Type: Defect
Fixes: https://github.com/vector-im/element-web/issues/23111

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Avoid hardcoding branding in user onboarding ([\#9206](https://github.com/matrix-org/matrix-react-sdk/pull/9206)). Fixes vector-im/element-web#23111. Contributed by @justjanne.

### CODE REVIEW

**Review by t3chguy — APPROVED** [review]

File: cypress/e2e/spotlight/spotlight.spec.ts [hunk 1]
```diff
@@ -296,7 +297,7 @@ describe("Spotlight", () => {
             cy.spotlightResults().eq(0).should("contain", room3Id);
         }));
     });
-    */
+    *//*
```
Comment (t3chguy) [inline cypress/e2e/spotlight/spotlight.spec.ts:300]: commenting out tests is not great, you can use `it.skip` but also disabling tests should be done in its own PR with a good reason.

**Review by t3chguy — CHANGES_REQUESTED** [review]

File: cypress/e2e/spotlight/spotlight.spec.ts (hunk 1 above)
Comment (justjanne) [reply cypress/e2e/spotlight/spotlight.spec.ts]: Oh f... thanks for catching that, that wasn’t even intended to be committed at all. I wish cypress-UI would allow running individual tests from the UI

Comment (t3chguy) [reply cypress/e2e/spotlight/spotlight.spec.ts]: @justjanne doesn't solve not having a UI but you can use `it.only` to run just a single test within a spec rather than skipping/commenting out the rest

**Review by t3chguy — APPROVED** [review]
